firmware/application/ui_external_items_menu_loader.cpp

```diff
@@ -0,0 +1,149 @@
+#include "ui_external_items_menu_loader.hpp"
+
+#include "sd_card.hpp"
+
+namespace ui {
+
+std::vector<Bitmap*> ExternalItemsMenuLoader::bitmaps;
+
+std::vector<GridItem> ExternalItemsMenuLoader::load_external_items(app_location_t app_location, NavigationView& nav) {
+    for (const auto& bitmap : bitmaps) {
+        delete bitmap->data;
```

Comment:
If you’re not going to use a smart pointer, you need to `delete[]` here because you allocated an array.

Reply:
Ah, nice catch